app/views/declaration/consignor_eori_number.scala.html

```diff
@@ -50,7 +50,7 @@
     backButton = Some(BackButton(messages("site.back"), Navigator.backLink(ConsignorEoriNumber, mode)))){
 
     @formHelper(action = controllers.declaration.routes.ConsignorEoriNumberController.submit(mode), 'autoComplete -> "off") {
-        @errorSummary(form.errors)
+        @errorSummary(radioGroupErrors("hasEori", "Yes", form.errors))
```

Comment:
Why it's not `code_yes`? Are we using something different for this page?

Reply:
Yes.  Not every page with a yes/no question is using the yesNoRadio component (and hence the code_yes, code_no ids).